src/app/modules/chip-filter/chip-filter.component.ts

```diff
@@ -155,6 +157,7 @@ export class AppChipFilterComponent implements ControlValueAccessor, OnInit, OnC
   }
 
   onFilterTextChange($event) {
+    this.focusItem = 2; // item 2: search textbox
```

Comment:
Maybe make the focusItem values constant variables if item 2 always represents the search textbox wherever this chip filter is used